src/Spd.Manager.Licence/ControllingMemberCrcAppManager.cs

```diff
@@ -280,20 +280,46 @@ private async Task<ChangeSpec> AddDynamicTasks(ControllingMemberCrcApplicationRe
         CancellationToken ct)
     {
         ChangeSpec changes = new();
+        //PeaceOfficerStatusChanged: check if Hold a Position with Peace Officer Status changed, create task with high priority, assign to Licensing CS team
+        PoliceOfficerRoleCode? originalRoleCode = originalApp.PoliceOfficerRoleCode == null ? null
+            : Enum.Parse<PoliceOfficerRoleCode>(originalApp.PoliceOfficerRoleCode.ToString());
 
+        if (newRequest.IsPoliceOrPeaceOfficer != originalApp.IsPoliceOrPeaceOfficer ||
+            newRequest.PoliceOfficerRoleCode != originalRoleCode ||
+            newRequest.OtherOfficerRole != originalApp.OtherOfficerRole ||
+            newFileInfos.Any(d => d.LicenceDocumentTypeCode == LicenceDocumentTypeCode.PoliceBackgroundLetterOfNoConflict))
+        {
+            IEnumerable<string> fileNames = newFileInfos.Where(d => d.LicenceDocumentTypeCode == LicenceDocumentTypeCode.PoliceBackgroundLetterOfNoConflict).Select(d => d.FileName);
+            changes.PeaceOfficerStatusChanged = true;
+            changes.PeaceOfficerStatusChangeTaskId = (await _taskRepository.ManageAsync(new CreateTaskCmd()
+            {
+                Description = $"Business have to requested to update below submitted information for its controlling member:" +
+                $"\nControlling Member - {originalApp.Surname + ", " + originalApp.GivenName}\nUpdated: Peace Officer Status\nFile Names {string.Join("\n* ", fileNames)} ",
+                DueDateTime = DateTimeOffset.Now.AddDays(3),
+                Subject = $"Updating controlling member background information for Case Number {originalApp.CaseNumber}",
```

Comment:
what case number? controlling member application case number or biz app case number?

Reply:
it is controlling member, would ask Kenny.